userpanel/style/bclean/style.less

```diff
@@ -28,17 +28,15 @@
 @import "../../../css/fontawesome/less/brands";
 @import "../../../css/fontawesome/less/regular";
 
-.alert,
-INPUT.alert,
-INPUT.alert:focus,
-TEXTAREA.alert,
-TEXTAREA.alert:focus,
+.lms-userpanel-alert,
+INPUT.lms-userpanel-alert,
+INPUT.lmsuserpanelui-alert:focus,
```

Comment:
Czy to celowe?

Reply:
Nie to mój błąd, ale teraz jestem offline do poniedziałku :)

Reply:
@kyob poprawiłem bezpośrednio w Twojej gałęzi.

1. Czy tak jak teraz będzie dobrze?
1. Czy problem dotyczy również `stable`?

Reply:
Może na wyrost napisałem bugfix. Bardziej o unikanie nadpisywania natywnych klas Bootstrapa.